json/src/test/java/com/basistech/rosette/dm/json/plain/JsonTest.java

```diff
@@ -491,7 +502,16 @@ public void oneWithEverything() {
         crListBuilder.add(topicResult);
         builder.topicResults(crListBuilder.build());
 
+        Embeddings.Builder embeddingsBuilder = new Embeddings.Builder();
+        EmbeddingCollection.Builder embeddingCollectionBuilder = new EmbeddingCollection.Builder();
+        embeddingCollectionBuilder.put(0, new float[]{0, 1});
+        embeddingsBuilder.put(Embeddings.TEXT, embeddingCollectionBuilder.build());
+        embeddings = embeddingsBuilder.build();
+        builder.embeddings(embeddingsBuilder.build());
+
```

Comment:
To actually retrieve a word vector for token index 3, I'd have to write this:

```
        float[] wordVector3 = text.getEmbeddings().get(Embeddings.TOKENS).getEmbeddings().get(3);
```

Notice we have two different methods named `getEmbeddings`.  Maybe the second one
should be `getEmbedding`, singular?  The embedding for a particular index.